scripts/release_info.py

```diff
@@ -285,26 +296,40 @@ def main():
             'This script should only be run from the latest release branch.')
 
     parsed_args = _PARSER.parse_args()
-    current_release = _get_current_version_tag()
-    base_commit = _get_base_commit_with_develop(current_release)
+    if parsed_args.personal_access_token is None:
+        print('No personal access token provided, please set up a personal '
+              'access token at https://github.com/settings/tokens and pass it '
+              'to the script using --personal_access_token=<token>')
+        return
+
+    personal_access_token = parsed_args.personal_access_token
+    g = github.Github(personal_access_token)
+    repo = g.get_organization('oppia').get_repo('oppia')
+
+    current_release = _get_current_version_tag(repo)
+    current_release_tag = current_release.name
+    base_commit = current_release.commit.sha
+    new_commits = get_extra_commits_in_new_release(base_commit, repo)
     new_release_logs = _gather_logs(base_commit)
+
+    for index, log in enumerate(new_release_logs):
+        is_cherrypicked = True
```

Comment:
(Optional) I think you can do:

is_cherrypicked = all([log.sha1 != commit.sha for commit in new_commits])